premake5.lua

```diff
@@ -118,14 +141,33 @@ project "Sandbox"
 		"%{IncludeDir.glm}"
 	}
 
-	links
```

Comment:
I think this shouldn't be removed. This would be place for a project links and under filter section has platform dependent links

Reply:
I agree. Fixed in https://github.com/TheCherno/Hazel/pull/82/commits/04a580604988cc4d33b82749c320d32c2f8c47f0.